packages/javascript-sdk/src/fr-webauthn/index.ts

```diff
@@ -279,8 +279,8 @@ abstract class FRWebAuthn {
   public static getAuthenticationOutcome(
     credential: PublicKeyCredential | null,
   ):
-    | OutcomeWithName<string, AttestationType, typeof credential>
-    | OutcomeWithName<string, AttestationType, typeof credential, string> {
+    | OutcomeWithName<string, AttestationType, PublicKeyCredential>
```

Comment:
caused issues in tsconfigs that do libchecks

Reply:
Why is there a leading pipe? Is this a new feature or a typo?